frame/merkle-mountain-range/src/mmr/storage.rs

```diff
@@ -58,14 +58,33 @@ impl<StorageType, T, I, L> Default for Storage<StorageType, T, I, L> {
 	}
 }
 
+fn parent_hash_of_ancestor_that_added_node<T: frame_system::Config>(
+	pos: NodeIndex,
+) -> <T as frame_system::Config>::Hash {
+	// TODO: turn this in a pallet storage item recording block-num when pallet was activated.
```

Comment:
IIUC it may be computed with something like `frame_system::BlockNumber - NumberOfLeaves`, assuming that we're addign exactly one leaf for every block. So storage isn't necessary here. At least I've been trying to do that in the light BEEFY client. I may be wrong though :)